wordpress-mobile/WordPress-iOS
# Reader Saved Posts: Display promo alert the first time the feature is used

This PR adds an informative alert the first time the user tries to use the new Reader Saved Posts feature (and replaces #9591, targeting `develop` instead):

![simulator screen shot - iphone 8 - 2018-06-15 at 22 26 22](https://user-images.githubusercontent.com/4780/41488946-69553180-70ee-11e8-8ee1-8fb0d7f7165c.png)

I will say I'm not super happy that we currently have to repeat the transitioning delegate everywhere we want to present a fancy alert at the moment, although I think I have an idea for how to improve that in the future.

**To test:**

* Visit the Reader
* Save a post and ensure you are shown the promo alert above
* Try to save another post. Ensure that the promo isn't shown a second time.

cc @iamthomasbishop
